lib/rules/no-duplicate-selectors/README.md

```diff
@@ -100,3 +100,22 @@ a {
   & c {}
 }
 ```
+
+## Optional secondary options
+
+### `disallowInList: true | false` (default: `false`)
+
+This option will dissallow duplication selectors within a group selectors
```

Comment:
```suggestion
This option will also disallow duplicate selectors within selector lists.
```

We like to keep the [vocab](http://apps.workflower.fi/vocabs/css/en) we use consistent. [Selector list](https://www.w3.org/TR/selectors-4/#grouping) comes from the spec.